src/SprykerEco/Shared/PunchoutCatalogs/Transfer/punchout_catalogs.transfer.xml

```diff
@@ -61,6 +62,11 @@
         <property name="isSuccessful" type="bool"/>
     </transfer>
 
+    <transfer name="PunchoutCatalogConnectionFilter">
```

Comment:
I thought we agreed to reuse existing `PunchoutCatalogConnectionCriteriaTransfer` and add 2 new props to it

Reply:
Oops, I thought that we eventually decided to use a separate transfer. I will rework it.